### PR TITLE
docs(cn): translate reference/react/useOptimistic into Chinese

### PR DESCRIPTION
**Pull request by Yucohny**
翻译 reference/react/useOptimistic。

### CODE REVIEW

**Review by awxiaoxian2020 — DISMISSED** [review]

File: src/content/reference/react/useOptimistic.md [hunk 1]
```diff
@@ -23,51 +23,51 @@ The `useOptimistic` Hook is currently only available in React's Canary and exper
...
+* `updateFn(currentState, optimisticValue)`：一个函数，接受当前 state 和传递给 `addOptimistic` 的乐观值，并返回结果乐观状态。它必须是一个纯函数。`updateFn` 接受两个参数：`currentState` 和 `optimisticValue`。返回值将是 `currentState` 和 `optimisticValue` 的合并值。
 
 
-#### Returns {/*returns*/}
+#### 返回值 {/*returns*/}
 
-* `optimisticState`: The resulting optimistic state. It is equal to `state` unless an action is pending, in which case it is equal to the value returned by `updateFn`.
-* `addOptimistic`: `addOptimistic` is the dispatching function to call when you have an optimistic update. It takes one argument, `optimisticValue`, of any type and will call the `updateFn` with `state` and `optimisticValue`.
+* `optimisticState`：结果乐观状态。除非有操作挂起，否则它等于 `state`，在这种情况下，它等于 `updateFn` 返回的值。
+* `addOptimistic`：触发乐观更新时调用的 dispatch 函数。它接受一个可以是任何类型的参数 `optimisticValue`，并将调用 `updateFn` 以 `state` 和 `optimisticValue` 作为参数。
```
Comment (loveloki) [inline src/content/reference/react/useOptimistic.md:60]:
```suggestion
* `addOptimistic`：触发乐观更新时调用的 dispatch 函数。它接受一个可以是任何类型的参数 `optimisticValue`，并以 `state` 和 `optimisticValue` 作为参数来调用 `updateFn`。
```

File: src/content/reference/react/useOptimistic.md [hunk 2]
```diff
@@ -23,51 +23,51 @@ The `useOptimistic` Hook is currently only available in React's Canary and exper
...
 ---
 
-## Usage {/*usage*/}
+## 用法 {/*usage*/}
 
-### Optimistically updating forms {/*optimistically-updating-with-forms*/}
+### 乐观地更新表单 {/*optimistically-updating-with-forms*/}
 
-The `useOptimistic` Hook provides a way to optimistically update the user interface before a background operation, like a network request, completes. In the context of forms, this technique helps to make apps feel more responsive. When a user submits a form, instead of waiting for the server's response to reflect the changes, the interface is immediately updated with the expected outcome.
+`useOptimistic` Hook 提供了一种在后台操作（如网络请求）完成之前乐观地更新用户界面的方式。在表单的上下文中，这种技术有助于使应用程序感觉更加响应。当用户提交表单时，界面立即更新为预期的结果，而不是等待服务器的响应来反映更改。
```
Comment (loveloki) [inline src/content/reference/react/useOptimistic.md:68]:
```suggestion
`useOptimistic` Hook 提供了一种在后台操作（如网络请求）完成之前乐观地更新用户界面的方式。在表单的上下文中，这种技术有助于使应用程序在感觉上响应地更加快速。当用户提交表单时，界面立即更新为预期的结果，而不是等待服务器的响应来反映更改。
```

File: src/content/reference/react/useOptimistic.md [hunk 3]
```diff
@@ -23,51 +23,51 @@ The `useOptimistic` Hook is currently only available in React's Canary and exper
...
 
-For example, when a user types a message into the form and hits the "Send" button, the `useOptimistic` Hook allows the message to immediately appear in the list with a "Sending..." label, even before the message is actually sent to a server. This "optimistic" approach gives the impression of speed and responsiveness. The form then attempts to truly send the message in the background. Once the server confirms the message has been received, the "Sending..." label is removed.
+例如，当用户在表单中输入消息并点击“发送”按钮时，`useOptimistic` Hook 允许消息立即出现在列表中，并带有“发送中……”标签，即使消息实际上还没有发送到服务器。这种“乐观”方法给人一种速度和响应能力的印象。然后，表单尝试在后台真正发送消息。一旦服务器确认消息已收到，“发送中……”标签就会被移除。
```
Comment (loveloki) [inline src/content/reference/react/useOptimistic.md:70]:
```suggestion
例如，当用户在表单中输入消息并点击“发送”按钮时，`useOptimistic` Hook 允许消息立即出现在列表中，并带有“发送中……”标签，即使消息实际上还没有发送到服务器。这种“乐观”方法给人一种快速和响应灵敏的印象。然后，表单在后台尝试真正发送消息。一旦服务器确认消息已收到，“发送中……”标签就会被移除。
```

File: src/content/reference/react/useOptimistic.md [hunk 4]
```diff
@@ -5,13 +5,13 @@ canary: true
...
 
-The `useOptimistic` Hook is currently only available in React's Canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+`useOptimistic` Hook 仅在 Canary 与 experimental 渠道中可用。参阅 [React 发布渠道](/community/versioning-policy#all-release-channels) 以了解更多信息。
 
 </Canary>
 
 <Intro>
 
-`useOptimistic` is a React Hook that lets you optimistically update the UI.
+`useOptimistic` 是一个 React Hook，它可以帮助更乐观地更新用户界面。
```
Comment (loveloki) [inline src/content/reference/react/useOptimistic.md:14]:
```suggestion
`useOptimistic` 是一个 React Hook，它可以帮助你更乐观地更新用户界面。
```

File: src/content/reference/react/useOptimistic.md [hunk 5]
```diff
@@ -23,51 +23,51 @@ The `useOptimistic` Hook is currently only available in React's Canary and exper
 
 ---
 
-## Reference {/*reference*/}
+## 参考 {/*reference*/}
 
 ### `useOptimistic(state, updateFn)` {/*use*/}
 
-`useOptimistic` is a React Hook that lets you show a different state while an async action is underway. It accepts some state as an argument and returns a copy of that state that can be different during the duration of an async action such as a network request. You provide a function that takes the current state and the input to the action, and returns the optimistic state to be used while the action is pending.
+`useOptimistic` 是一个 React Hook，它允许你在异步操作进行时显示不同 state。它接受 state 作为参数，并返回该 state 的副本，在异步操作（如网络请求）进行期间可以不同。你需要提供一个函数，该函数接受当前 state 和操作的输入，并返回在操作挂起期间要使用的乐观状态。
```
Comment (loveloki) [inline src/content/reference/react/useOptimistic.md:30]:
```suggestion
`useOptimistic` 是一个 React Hook，它允许你在进行异步操作时显示不同 state。它接受 state 作为参数，并返回该 state 的副本，在异步操作（如网络请求）期间可以不同。你需要提供一个函数，该函数接受当前 state 和操作的输入，并返回在操作挂起期间要使用的乐观状态。
```

**Comment by loveloki** [comment]
我愿称之为 `useLoading`

**Review by loveloki — APPROVED** [review]

**Pull request merged**